cla-backend-go/events/repository.go

```diff
@@ -566,20 +566,18 @@ func (repo *repository) GetCompanyFoundationEvents(companySFID, companyID, found
 }
 
 // GetCompanyClaGroupEvents returns the list of events for cla group and the company
-func (repo *repository) GetCompanyClaGroupEvents(clagroupID string, companySFID string, nextKey *string, paramPageSize *int64, searchTerm *string, all bool) (*models.EventList, error) {
+func (repo *repository) GetCompanyClaGroupEvents(projectSFID string, companySFID string, nextKey *string, paramPageSize *int64, searchTerm *string, all bool) (*models.EventList, error) {
 	f := logrus.Fields{
 		"functionName":  "v1.events.repository.GetCompanyClaGroupEvents",
-		"claGroupID":    clagroupID,
```

Comment:
This method actually Gets Company CLA Group Events and not Company Project Events. Which use case would this be for ?

Reply:
This method used for GetCompanyProjectEventsHandler only, Where we are getting projectsfID, So i thought it's best fitted to pass projectSFID in order to use index: CompanySFIDProjectIDEpochIndex.

Reply:
So the use case is to aggregate projectSFID events under a given CLA Group @umeshlumbhani247